test/powershell/Modules/Microsoft.PowerShell.Utility/Invoke-Item.Tests.ps1

```diff
@@ -59,6 +59,31 @@ Describe "Invoke-Item basic tests" -Tags "CI" {
         }
         Get-Content $redirectFile -Raw | Should Match "usage: ping"
     }
+
+    It "Should invoke a folder without error" {
+        if ($IsWindows)
+        {
+            $shell = New-Object -ComObject "Shell.Application"
+            $windows = $shell.Windows()
+
+            $before = $windows.Count
+            Invoke-Item -Path $PSHOME
+            # give it time to open
+            Start-Sleep -Milliseconds 500
+            $after = $windows.Count
+
+            $before + 1 | Should Be $after
+            $item = $windows.Item($after - 1)
+            $item.LocationURL | Should Match ($PSHOME -replace '\\', '/')
+            ## close the windows explorer
+            $item.Quit()
+        }
+        else
+        {
+            # can't validate that it actually opened, but no error should be returned
+            Invoke-Item -Path $PSHOME
```

Comment:
Not able to figure out correct command to get this to work.  It looks like `lsof` only works with files and not directories.  If you specify a directory, it's looking for any files within that directory and not the directory itself.  Also, if I get Ubuntu GUI to open $pshome directory, it doesn't show up in `lsof` output at all.

```bash
lsof +d $pshome | grep -v "^powershel"
```